packages/notifier/test/test-makeNotifierFromSubscriber.js

```diff
@@ -331,11 +348,11 @@ test('makeNotifierFromSubscriber - getUpdateSince() result identity', async t =>
   );
 
   const { publisher, subscriber: failureSubscriber } = makePublishKit();
-  const failureNotifier = await makeNotifierFromSubscriber(failureSubscriber);
+  const failureNotifier = makeNotifierFromSubscriber(failureSubscriber);
   publisher.publish('first value');
   ({ updateCount } = await failureNotifier.getUpdateSince());
-  const failureP = failureNotifier.getUpdateSince();
-  const failureP2 = failureNotifier.getUpdateSince();
+  const failureP = failureNotifier.getUpdateSince(updateCount);
+  const failureP2 = failureNotifier.getUpdateSince(updateCount);
```

Comment:
is this a better test or a change in semantics?

Reply:
Thanks for catching this.  An intermediate commit had a semantics change, but I shouldn't have papered it over.  Restoring the original test shows that the latest version has no semantic change.